File: LocalHiringPlatform.Domain/Models/Experience/CandidateExperienceCreateModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LocalHiringPlatform.Domain.Models.Experience
{
    public class CandidateExperienceCreateModel
    {
        public Guid? EntityId { get; set; }

        public string CompanyName { get; set; } = string.Empty;

        public string Designation { get; set; } = string.Empty;

        public int IndustryTypeId { get; set; }

        public string City { get; set; } = string.Empty;

        public string State { get; set; } = string.Empty;

        public string Country { get; set; } = string.Empty;

        public DateOnly StartDate { get; set; }

        public DateOnly? EndDate { get; set; }

        public bool IsCurrentCompany { get; set; }

        public string Summary { get; set; } = string.Empty;
    }
}

File: UI/localhiringplatform-ui/src/types/Experience/CandidateExperienceCreateModel.ts
export interface CandidateExperienceCreateModel {

    entityId?: string;

    companyName: string;

    designation: string;

    industryTypeId: number;

    city: string;

    state: string;

    country: string;

    startDate: string;

    endDate?: string;

    isCurrentCompany: boolean;

    summary: string;
}